test/FastTests/Issues/RavenDB_16590.cs

```diff
@@ -73,7 +73,11 @@ public void StudioListsOfIndexScopedSettings()
                 "Indexing.TimeSinceLastQueryAfterWhichDeepCleanupCanBeExecutedInMin",
                 "Indexing.Throttling.TimeIntervalInMs",
                 "Indexing.TransactionSizeLimitInMb",
-                "Indexing.Static.SearchEngineType"
+                "Indexing.Static.SearchEngineType",
+                "Indexing.OrderByScoreAutomaticallyWhenBoostingIsInvolved",
+                "Indexing.Lucene.UseCompoundFileInMerging",
+                "Indexing.UseCompoundFileInMerging",
```

Comment:
It should be `Indexing.Lucene.UseCompoundFileInMerging`

Reply:
Not here, this test gets all key form server check if we handled them manually.